### ISSUE TITLE
Tribool used for --reauthentication option showed in the printed help

### ISSUE THREAD

**Issue by ppatierno** [redhat-developer/app-services-cli#1394]
When asking for the help of the `rhoas kafka update` command, we get the following about the `--reauthentication` flag.

```shell
--reauthentication Tribool   Enable connection reauthentication for the Kafka instance
```

I think that "Tribool" string is really confusing assuming that user can chose just between `true` and `false` if he wants re-authentication enabled or not on his Kafka instance.
I checked the code but not sure why a new `Tribool` type was used for that option, maybe any good reason I didn't get but anyway I think that we should avoid print it in the help. Unless user can specify "" as third possible value? For which reason?

**Comment by wtrocki** [comment]
I agree. Tribool is not something universally known. Tribool is actually implementation detail of go that allows empty value.
We going to fix that - I think boolean is self explanatory.

**Comment by ppatierno** [comment]
@wtrocki thanks but what does the empty value mean in this case regarding a configuration parameter which can be just true or false, just curious about of it.

**Comment by wtrocki** [comment]
Command sets various configurations. Empty value means:

> Please do not change this value on backend  

this is also default value

**Comment by ppatierno** [comment]
but if I am updating some Kafka instance info and I don't want to change the `reauthentication` as it is currently set on backend I can just not specify it ... or I don't get what you mean :-)

> this is also default value

I guess that you mean default value for the CLI itself that when talks to the control plane doesn't specify any reauthentication or just empty so the control plane creates the Kafka instance with the default one which is actually "true". In the end, real default is "true", it's not empty. As I said empty default is in terms of contract between CLI and control plane ... it still doesn't make any sense to me from a user point of view who wants re-authentication enabled or not.
What am I missing here? :-(

**Comment by rkpattnaik780** [comment]
Hi @ppatierno 
We had to introduce tribool as omitting the flag still mapped to false (which could make things complex when user omits the `--reauthentication` flag when the re authentication is enabled for the kafka instance). A tribool allowed us to detect if flag has been set deliberately. 
I agree tribool looks confusing in the docs, we will get to a workaround and remove it.

**Comment by wtrocki** [comment]
To be correct - we only want to remove tribool from docs and replace it with bool. Is that right
